src/main/java/com/uid2/shared/secure/azurecc/MaaTokenSignatureValidator.java

```diff
@@ -65,7 +66,7 @@ public MaaTokenPayload validate(String tokenString) throws AttestationException
                 tokenVerifier.verify(tokenString);
             }
         } catch (TokenVerifier.VerificationException e) {
-            throw new AttestationException("Fail to validate the token signature, error: " + e.getMessage());
+            throw new AttestationClientException("Fail to validate the token signature, error: " + e.getMessage());
```

Comment:
Does this message get returned to the caller? What does it contain?

Reply:
It will return to caller. No confidential data will be exposed.

Error message could be:
- token expired
- audience not match
- algorithm not supported
- keyid not found
- signature validation failed
- ...

<img width="1188" alt="image" src="https://github.com/IABTechLab/uid2-shared/assets/137876002/a439e80f-1838-43c1-bcc6-cf35b2e4dd55">